pkg/scaling/executor/scale_jobs.go

```diff
@@ -101,6 +101,10 @@ func (e *scaleExecutor) getScalingDecision(scaledJob *kedav1alpha1.ScaledJob, ru
 }
 
 func (e *scaleExecutor) createJobs(ctx context.Context, logger logr.Logger, scaledJob *kedav1alpha1.ScaledJob, scaleTo int64, maxScale int64) {
+	if maxScale <= 0 {
+		logger.Info("No need to create jobs", "Effective maxScale", maxScale)
```

Comment:
Can we extend this message a bit? the `Effective maxScale` part, it might not be clear to everyone